frontend/ui/widget/dictquicklookup.lua

```diff
@@ -542,6 +542,8 @@ function DictQuickLookup:init()
                 },
             })
         end
+--        comment the next line if you don't want the hyphenation button in dictquicklookup
+        self.ui.userhyph:addButton(buttons, 3, self.word, self)
```

Comment:
Users won't be able to comment that line :)
We need to decide if we still need to have that below all dict results (or if you just needed that for quicker devel).